ballerina/modules/oas/types.bal

```diff
@@ -0,0 +1,8797 @@
+    *http:Ok;
+    FulfillmentOrder body;
+|};
+
+public type BadRequestInlineResponse400 record {|
+    *http:BadRequest;
+    InlineResponse400 body;
+|};
+
+public type OkRejectFulfillmentResponse record {|
+    *http:Ok;
+    RejectFulfillmentResponse body;
+|};
+
+public type UnprocessableEntityInlineResponse42211 record {|
+    *http:UnprocessableEntity;
+    InlineResponse42211 body;
+|};
+
+public type UnprocessableEntityInlineResponse42210 record {|
+    *http:UnprocessableEntity;
+    InlineResponse42210 body;
+|};
+
+public type UnprocessableEntityInlineResponse42215 record {|
+    *http:UnprocessableEntity;
+    InlineResponse42215 body;
+|};
+
+public type UnprocessableEntityInlineResponse42214 record {|
+    *http:UnprocessableEntity;
+    InlineResponse42214 body;
+|};
+
+public type UnprocessableEntityInlineResponse42213 record {|
+    *http:UnprocessableEntity;
+    InlineResponse42213 body;
+|};
+
+public type UnprocessableEntityInlineResponse42212 record {|
+    *http:UnprocessableEntity;
+    InlineResponse42212 body;
+|};
+
+public type OkTransitionFulfillmentOrder record {|
+    *http:Ok;
+    TransitionFulfillmentOrder body;
+|};
+
+public type AcceptedCompleteCheckoutResponse record {|
+    *http:Accepted;
+    CompleteCheckoutResponse body;
+|};
+
+public type UnprocessableEntityInlineResponse42218 record {|
+    *http:UnprocessableEntity;
+    InlineResponse42218 body;
+|};
+
+public type UnprocessableEntityInlineResponse42217 record {|
+    *http:UnprocessableEntity;
+    InlineResponse42217 body;
+|};
+
+public type OkCalculateRefund record {|
+    *http:Ok;
+    CalculateRefund body;
+|};
+
+public type UnprocessableEntityInlineResponse42216 record {|
+    *http:UnprocessableEntity;
+    InlineResponse42216 body;
+|};
+
+public type OkAcceptFulfillmentResponse record {|
+    *http:Ok;
+    AcceptFulfillmentResponse body;
+|};
+
+public type AcceptedModifyDraftOrder record {|
+    *http:Accepted;
+    ModifyDraftOrder body;
+|};
+
+public type NoContentString record {|
+    *http:NoContent;
+    string body;
+|};
+
+public type OkCancelFulfillmentOrder record {|
+    *http:Ok;
+    CancelFulfillmentOrder body;
+|};
+
+public type OkResourceFeedbackList record {|
+    *http:Ok;
+    ResourceFeedbackList body;
+|};
+
+public type AcceptedShippingRates record {|
+    *http:Accepted;
+    ShippingRates body;
+|};
+
+public type OkInlineResponse20062 record {|
+    *http:Ok;
+    InlineResponse20062 body;
+|};
+
+public type OkMoveFulfillmentOrderResponse record {|
+    *http:Ok;
+    MoveFulfillmentOrderResponse body;
+|};
+
+public type AcceptedCreateAuthorizationResponse record {|
+    *http:Accepted;
+    CreateAuthorizationResponse body;
+|};
+
+public type OkMarkCommentResponse record {|
+    *http:Ok;
+    MarkCommentResponse body;
+|};
+
+public type ConflictInlineResponse409 record {|
+    *http:Conflict;
+    InlineResponse409 body;
+|};
+
+public type NotFoundInlineResponse404 record {|
```

Comment:
Why are these status code records in the `types.bal`? Generating only the client should not create these types